TLM/TLM/UI/Helpers/DLCRestrictedCheckboxOption.cs

```diff
@@ -0,0 +1,59 @@
+namespace TrafficManager.UI.Helpers;
+
+using System;
+using ColossalFramework.Globalization;
+using ColossalFramework.UI;
+using ICities;
+using TrafficManager.State;
+using UnityEngine;
+
+public class DLCRestrictedCheckboxOption : CheckboxOption {
+    private SteamHelper.DLC _requiredDLC;
+
+    public DLCRestrictedCheckboxOption(string fieldName,
+                                       SteamHelper.DLC requiredDLC,
+                                       Options.PersistTo scope = Options.PersistTo.Savegame) : base(fieldName, scope) {
+        _requiredDLC = requiredDLC;
+        _readOnly = !SteamHelper.IsDLCOwned(_requiredDLC);
+    }
+
+    public override CheckboxOption AddUI(UIHelperBase container) {
+        UIPanel panel = ((UIPanel)((UIHelper)container).self).AddUIComponent<UIPanel>();
+        panel.autoLayout = false;
+        panel.size = new Vector2(720, 22);//default checkbox template size
+        panel.relativePosition = Vector3.zero;
```

Comment:
not important but I think the code works the same without these lines.

Reply:
It does not. Default size of panel is 600x900 or something like that and is not added as first panel (rel. 0,0)
Notice I added panel and then controls to the panel to be able to shift everything imitating negative left margin.